BMM.UI.Android/Application/Fragments/PlayerFragment.cs

```diff
@@ -89,6 +92,12 @@ public void OnStopTrackingTouch(SeekBar seekBar)
             ViewModel.IsSeeking = false;
         }
 
+        public override void OnCreateOptionsMenu(IMenu menu, MenuInflater inflater)
+        {
+            base.OnCreateOptionsMenu(menu, inflater);
+            inflater.Inflate(Resource.Menu.playback_history, menu);
```

Comment:
isn't the playback history on the home screen? Why does the PlayerFragment have anything to do with it?

Reply:
Because it opens exactly after home screen fragment and overrides existing options menu. But now I found some weird code in BaseFragment, so I think I'll do it other way.